### PR TITLE
WIP to debug wrong arch of native node addons in mac universal build for x64

### PR DESCRIPTION
**Pull request by csett86**
- chore: package-lock.json noise
- 2021.11.3

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: '16'
+    - name: Install dependencies
+      run: sudo apt-get install libx11-dev zlib1g-dev libpng-dev libxtst-dev
```
Comment (saghul) [inline .github/workflows/ci.yml:23]: Why is needed again?

Comment (csett86) [reply .github/workflows/ci.yml:23]: We currently have the very unfortunate situation that local mac os builds on mac os 11 work fine for both archs, but the builds using the same prebuilds in gh actions fail to launch on intel macs. 

This pr is not meant to be merged, but I am using it to trying to get to the bottom of this issue and the provide a proper fix.

File: package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "generateUpdatesFilesForAllChannels": true,
     "afterPack": "./linux-sandbox-fix.js",
     "afterSign": "./notarize.js",
+    "buildDependenciesFromSource": true,
```
Comment (csett86) [inline package.json:26]: This is also why I added this line to test how gh actions behave if prebuilds are disabled in electron-builder

**Comment by csett86** [comment]
I will later add artifact archiving to the gh action so that I dont need to misuse a draft release to get to the build artifacts

File: .github/workflows/ci.yml (hunk 1 above)
Comment (saghul) [reply .github/workflows/ci.yml:23]: So the current release if broken on Intel Macs running macOS 11?

Comment (csett86) [reply .github/workflows/ci.yml:23]: No, our current release works fine, as our current release predates the prebuild changes. But current master build with GH actions is broken on intel macs.

Downstream from ffmuc.net ran into this when releasing based on current master: https://github.com/freifunkMUC/jitsi-meet-electron/issues/38

Comment (saghul) [reply .github/workflows/ci.yml:23]: Gotcha, thanks for looking into this!

**Pull request closed without merging**